src/graphql/fragments/loanCardFields.graphql

```diff
@@ -59,4 +59,9 @@ fragment loanCardFields on LoanBasic {
 	...on LoanDirect {
 		trusteeName
 	}
+
+	# for loan-progress component
+	unreservedAmount @client
+	fundraisingPercent @client
+	fundraisingTimeLeft @client
```

Comment:
This is a shared fragment and will affect all loan cards that use it so let's please not add this here at this time.

Also, we should have in place an animation on the progress bars so that these fields could always be dynamic and updated in the client.

Reply:
Yeah... I wasn't sure if we cared about adding these to the shared. But I'll move them.

And for the loading, it's more about what data is prefetched and what data isn't. The prefetched data didn't have these three props. I find an alternative path.

Reply:
oh well, that's actually a good point. If the "actual" graphql fields are already included in the loanFragment then the data has been fetched and these would just add the local field processing... I don't know how much additional latency these local fields might add...

Reply:
Thoughts on keeping on the shared then? I assumed it wasn't a huge impact. Otherwise I'll need to tweak the channel query or add a "client-side loading" flag to the new loan card. I don't mind doing those alternatives, if needed.

Reply:
I have an alternate solution now that checks those local resolver values for `undefined`. Happy to go either path, but I'll push up the alternative after walking the dog...

Reply:
It may be that it turns out to be ok to make this change but I think we should separate it and look at how the other queries that implement this fragment might handle the initial presence of these fields.

I'm guessing having local fields in a fragment worked fine during dev but that may be one aspect we want to further verify as well...

Reply:
Yeah the local fields in fragment worked fine, but I've just pushed up the alternate solution, pushing the data pull back to clientside, but adding some extra checks + loaders. Turns out the loaders for progress weren't working before, so these were good changes to get in regardless.

![image](https://user-images.githubusercontent.com/16867161/218626697-bfe96269-f78a-42e4-ae04-e3b44661f4da.png)